views/index.pug

```diff
@@ -0,0 +1,7 @@
+doctype html
+html(lang="en")
+  head
+    title Form #{method}
+  body
+    p Form with method set to #{method}
+    block content
```

Comment:
Is this file only included for templating? Seems excessive for the specs but not wrong.